src/GafferImage/Unpremultiply.cpp

```diff
@@ -85,11 +97,32 @@ void Unpremultiply::affects( const Gaffer::Plug *input, AffectedPlugsContainer &
 
 void Unpremultiply::hashChannelData( const GafferImage::ImagePlug *output, const Gaffer::Context *context, IECore::MurmurHash &h ) const
 {
-	std::string alphaChannel = alphaChannelPlug()->getValue();
+	std::string alphaChannel;
+	ConstStringVectorDataPtr inChannelNamesPtr;
+	bool ignoreMissingAlpha;
 
-	ChannelDataProcessor::hashChannelData( output, context, h );
+	{
+		ImagePlug::GlobalScope c( context );
+		alphaChannel = alphaChannelPlug()->getValue();
```

Comment:
It's theoretically possible that using a GlobalScope here could break a script where someone used `${image:channelName}` to choose an alpha channel on the fly. But we already don't allow that for Premultiply and my guess would be this is a theoretical rather than practical concern. Just mentioning it in case anyone wants to make the case for holding this part back to 1.5.

Reply:
Murray pointed out that it's probably fine to just retarget the whole thing to 1.5. I've done that, and updated the Changes to note this change in behaviour.